src/main/resources/liquibase/harness/change/changelogs/postgresql/changeCompositeTypeSchema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "1",
+        "author": "itTest",
+        "changes": [
+          {
+            "createCompositeType": {
+              "name": "myTypeForSchemaTest",
+              "typeAttributes": [
+                {
+                  "typeAttribute": {
+                    "name": "attr1",
+                    "type": "integer"
+                  }
+                },
+                {
+                  "typeAttribute": {
+                    "name": "attr2",
+                    "type": "text",
+                    "collation": "en_US"
+                  }
+                }
+              ]
+            }
+          }
+        ],
+        "rollback": {
+          "changes": [
+            {
+              "sql": {
+                "sql": "DROP TYPE IF EXISTS public.\"myTypeForSchemaTest\";"
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "changeSet": {
+        "id": "2",
+        "author": "itTest",
+        "changes": [
+          {
+            "sql": {
+              "sql": "CREATE SCHEMA IF NOT EXISTS test_schema;"
+            }
+          }
+        ],
+        "rollback": {
+          "changes": [
+            {
+              "sql": {
+                "sql": "DROP SCHEMA IF EXISTS test_schema;"
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "changeSet": {
+        "id": "3",
+        "author": "itTest",
+        "changes": [
+          {
+            "changeCompositeTypeSchema": {
+              "typeName": "myTypeForSchemaTest",
+              "oldSchemaName": "public",
+              "newSchemaName": "test_schema"
+            }
+          }
+        ],
+        "rollback": {
+          "changes": [
+            {
+              "sql": {
+                "sql": "ALTER TYPE test_schema.\"myTypeForSchemaTest\" SET SCHEMA public;"
+              }
+            }
+          ]
+        }
+      }
+    },
+    {
+      "changeSet": {
+        "id": "4",
+        "author": "itTest",
+        "changes": [
+          {
+            "sql": {
```

Comment:
results of the query from `sql` changeType are not returned, so it doesn't check anything  - changeset can be removed